ui/src/components/easter-2024/EasterStory.js

```diff
@@ -0,0 +1,123 @@
+import {
+  Box,
+  Button,
+  Container,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  useDisclosure,
+  Text,
+} from '@chakra-ui/react';
+import Stories from 'react-insta-stories';
+import PlayCircleIcon from '@mui/icons-material/PlayCircle';
```

Comment:
for this, can make use of this package (https://react-icons.github.io/react-icons/icons/md/) instead of a new one. the play button icon is MdOutlinePlayCircleFilled